### PR TITLE
Add application progress helpers

### PR DESCRIPTION
**Pull request by convolu**
We need higher level functions for handling the newly introduced challenge groups. 

This modules aims to do this gradually

### CODE REVIEW

**Comment by convolu** [comment]
@tombettany @radujipa @pazdera @alex5imon Could you please review?

File: kano_profile/app_progress_helpers.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Implement higher level progress helpers
+
+from kano_profile.apps import load_app_state_variable
+
+
+def has_completed_challenges(app_name):
```
Comment (alex5imon) [inline kano_profile/app_progress_helpers.py]: The name should describe a bit better the functionality. Perhaps `has_completed_a_challenge`?

Comment (convolu) [inline kano_profile/app_progress_helpers.py]: I will rename it to `has_completed_any_challenges`

**Comment by convolu** [comment]
I have updated the name of the function to something more explanatory

**Comment by alex5imon** [comment]
Please merge :+1:

**Pull request merged**